packages/lib/src/interactions/SelectToZoom.tsx

```diff
@@ -58,7 +58,7 @@ function SelectToZoom(props: Props) {
     <SelectionTool
       id="SelectToZoom"
       transform={computeZoomSelection}
-      validate={({ html }) => Box.fromPoints(...html).hasMinSize(minZoom)}
+      validate={({ html }) => html[0].manhattanDistanceTo(html[1]) >= minZoom}
```

Comment:
I did not rename the prop to avoid a breaking change. Does the name still make sense?

Reply:
Looks ok to me !